build/metadata/github-actions/README.md

```diff
@@ -83,19 +83,20 @@ Once your workflow has been created, the best way to confirm that the workflow i
 
 If the action fails, there may be a problem with your configuration. To investigate, dig into the action's logs to view any error messages.
 
-<!-- action-docs-inputs -->
+<!-- action-docs-inputs source="action.yml" -->
```

Comment:
The `action-docs` helper wasn't working without this, so that's why the diff in this file is a bit bigger than just the new subdirectory input.